### ISSUE TITLE
pnpm-lock.yaml is no longer updated

### ISSUE THREAD

**Issue by StarpTech** [ghiscoding/lerna-lite#234]
### Describe the bug

See [CI Job](https://github.com/wundergraph/wundergraph/runs/7114023092?check_suite_focus=true)
The command is executed but the updated lock file was not committed https://github.com/wundergraph/wundergraph/commit/b9dea13cbaa4cf33c9d7d2af48ca71bad8723f24

### Expectation

I miss the line

```
 git add -- packages/sdk/package.json packages/sdk/CHANGELOG.md packages/nextjs/package.json packages/nextjs/CHANGELOG.md pnpm-lock.yaml
```

### Reproduction

No idea

### Lerna config and logs

<details><summary>lerna.json</summary><p>

```json
<!-- Please paste your `lerna.json` here -->
```
</p></details>

<details><summary>lerna-debug.log</summary><p>

```txt
<!-- If you have a `lerna-debug.log` available, please paste it here -->
<!-- Otherwise, feel free to delete this <details> block -->
```
</p></details>

### Environment Info

```shell
| Executable        | Version |
| ----------------- | ------- |
| `lerna --version` | 1.5.1 |
| `npm --version`   | VERSION |
| `yarn --version`  | VERSION |
| `node --version`  | VERSION |

---
OR simply run `npx lerna info` command and copy+paste the result here
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/ghiscoding/lerna-lite/blob/master/.github/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://github.com/ghiscoding/lerna-lite#readme).
- [X] Check that there isn't [already an issue](https://github.com/ghiscoding/lerna-lite/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/ghiscoding/lerna-lite/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ghiscoding** [comment]
yeah that is not Lerna-Lite, pnpm broke it with their last release, see PR #233 where I tested couple things and pnpm comment I left [here](https://github.com/pnpm/pnpm/issues/4910#issuecomment-1169139541). I only identified it yesterday through my unit tests but in some ways I'm glad you actually confirmed the issue

**Comment by StarpTech** [comment]
@ghiscoding thank you for the fast feedback and fix. What's the purpose of `--lockfile-only` when the command does not update the lock file? :confused:  Maybe, I don't understand the difference between `--lockfile-only` and `--fix-lockfile`. I like your approach to calling both. Nothing could go wrong.

**Comment by ghiscoding** [comment]
The `--fix-lockfile` works because oh how the pnpm change was applied in the [PR](https://github.com/pnpm/pnpm/pull/4932) that I referenced, if you look at the changes [here](https://github.com/pnpm/pnpm/pull/4932/files) you see that he removed the `--lockfile-only` to make it exist early if no changes is detected (that is the breaking change for us) and if you look at the remaining of that `if` condition that he modified then you can see that `--fix-lockfile` is still part of the condition (previous behavior) and so I decided to give it a try and saw that it does work to get my unit tests working again in my PR #233, basically it's a bypass that I found for the new regression bug in pnpm to get their previous behavior, I don't like it but it's a quick fix until the regression is addressed (hopefully).

I'll look at releasing a new version later tonight including this patch fix (PR #233), but I still hope that pnpm revert their change or fix their regression eventually in a better way

**Comment by StarpTech** [comment]
Nice :+1:

**Comment by ghiscoding** [comment]
@StarpTech  by the way, my unit tests only failed in the CI but they seem to pass locally even with latest pnpm version. Was your new release done via CI or from your local computer? I'd like to know so that I can update the pnpm issue with relevant info. Thanks

**Comment by ghiscoding** [comment]
ok so it seems to fail only in CI mode so far, by the way to make it clear the quick fix is to use `pnpm install --lockfile-only --fix-lockfile` so it's now 2 flags instead of just 1, I think I didn't explain well earlier.

**Comment by ghiscoding** [comment]
release quick fix under [v1.6.0](https://github.com/ghiscoding/lerna-lite/releases/tag/v1.6.0) even though pnpm is expected to release a fix for the regression within the next day. Also note that pnpm maintainer gave me another alternative instead of the fix lock which is `pnpm install --lockfile-only --no-frozen-lockfile`
